src/java.base/windows/native/libjava/java_props_md.c

```diff
@@ -128,21 +128,30 @@ getEncodingInternal(LCID lcid)
     return ret;
 }
 
-static char* getConsoleEncoding()
+static char* getConsoleEncoding(BOOL output)
```

Comment:
I assume the "output" param is to prepare for introducing stdin.encoding in the future.

Reply:
Yes